README.md

```diff
@@ -1 +1,11 @@
-# hw01
\ No newline at end of file
+# hw01
```

Comment:
Everything looks really good! You could have maybe included some more elements, like lists or different header sizes, in order to convey more information.